src/main/java/com/rakovets/course/java/core/practice/arrays/Task02.java

```diff
@@ -43,9 +52,13 @@ static double getAverageMark(int[][] marks) {
      * @return минимальная отметка
      */
     static int getMinMark(int[][] marks) {
-        //TODO
-        // Код, решающий задачу пишем ниже, при этом используя параметры метода
-        return 0;
+        int min = Integer.MAX_VALUE;
+        for (int i = 0; i < marks.length; i++) {
+            for (int j = 0; j < marks[i].length; j++) {
+                min = Math.min(min, marks[i][j]);
+                }
+            }
```

Comment:
"Violation: There aren't correct indentations for inner code
Solution: Add correct indentations for inner code"